itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1607,6 +1607,40 @@ public void testTruncateWithCM() throws IOException {
   }
 
   @Test
+  public void testTruncateTempTable() throws IOException {
```

Comment:
I just added in TestReplicationScenarios.java as it was an impact due to changes for replicating truncate table.
But, I think your point is valid. Will share another patch with this change.